Update testing.c on runoff

curiouskiwi opened the pull request:
Changed votes in case 18 to be realistic (with voter_count set to 28, the tied votes must add to 28.)

curiouskiwi committed:
Update testing.c

Changed votes in case 18 to be realistic (with voter_count set to 28, the tied votes must add to 28.)

CarterZenke committed:
ensure total candidate votes amount to `voter_count`

CarterZenke commented:
Thanks, @curiouskiwi, I also added a615e8d to ensure other checks meet the same requirement. The staff solution passes this updated version, but I want to give you a chance to review before we merge!

curiouskiwi committed:
Update testing.c

tweak case 21, since candidate[0] could not have been already eliminated with 7 votes, dropped it to 5 and added the extra 2 to candidate[1] to maintain total votes.

curiouskiwi commented:
@CarterZenke I tweaked that case 21 check to make it realistic (candidate 0 with 5 votes, so it makes sense that they were already eliminated   and the extra 2 votes to candidate 1 to maintain the total votes count.

Verified that the staff version still passes all checks.
